tools/expotools/src/commands/UpdateReactNativeDocs.ts

```diff
@@ -0,0 +1,331 @@
+import chalk from 'chalk';
+import fs from 'fs-extra';
+import inquirer from 'inquirer';
+import path from 'path';
+
+import { GitDirectory } from '../Git';
+import logger from '../Logger';
+import { Directories } from '../expotools';
+
+type Options = {
+  sdk: string;
+  from: string;
+  to: string;
+};
+
+type DocsSummary = {
+  removed: string[];
+  added: string[];
+  changed: string[];
+};
+
+const EXPO_DIR = Directories.getExpoRepositoryRootDir();
+const DOCS_DIR = path.join(EXPO_DIR, 'docs');
+const SDK_DOCS_DIR = path.join(DOCS_DIR, 'pages', 'versions');
+
+const RN_REPO_DIR = path.join(DOCS_DIR, 'react-native-website');
+const RN_WEBSITE_DIR = path.join(RN_REPO_DIR, 'website');
+const RN_DOCS_DIR = path.join(RN_REPO_DIR, 'docs');
+
+const PREFIX_ADDED = 'ADDED_';
+const PREFIX_REMOVED = 'REMOVED_';
+const SUFFIX_CHANGED = '.diff';
+
+const DOCS_IGNORED = [
```

Comment:
can we generate this from looking at our docs directory? so if a file existed in both `from` and `to` but does not exist in the docs directory then we consider it ignored.

don't have a strong opinion about it, but the benefit would be that we wouldn't have to manually maintain a list

Reply:
Yeah, that might be possible. I have to check the exact format patches, but basically:

> If diff adds file
>   - new file added (not ignored)
> else
>   - file is ignored

Reply:
i think it's not a blocker for this pr but something to follow up with on afterwards maybe

Reply:
Ok, let's merge this one for now to at least unblock the RN versions update. I'll add more PRs fine-tuning the behavior with the tips from this PR! The 3 PRs in addition I'll be working on:
- [ ] Tweak the Expo Tools Github wrapper to allow empty options
- [ ] Auto-detect ignored files from updated version range
- [ ] Optionally, tweak the github range input with cursors